src/data/list/perm.lean

```diff
@@ -39,10 +39,10 @@ trans (swap _ _ _) (skip _ $ skip _ p)
 
 attribute [trans] perm.trans
 
-theorem perm.eqv (α : Type) : equivalence (@perm α) :=
+theorem perm.eqv (α : Type uu) : equivalence (@perm α) :=
 mk_equivalence (@perm α) (@perm.refl α) (@perm.symm α) (@perm.trans α)
 
-instance is_setoid (α : Type) : setoid (list α) :=
+instance is_setoid (α : Type uu) : setoid (list α) :=
```

Comment:
Why not use `Type*`, or just nothing at all, i.e. `(α)`?

Reply:
Because `uu` was declared upstairs...